src/object/portable.hpp

```diff
@@ -33,14 +33,17 @@ class Portable : public GameObjectComponent
   virtual ~Portable() {}
 
   /** called each frame when the object has been grabbed. */
-  virtual void grab(MovingObject& other, const Vector& pos, Direction dir) = 0;
-  virtual void ungrab(MovingObject& other, Direction ) {}
+  virtual void grab(MovingObject& other, const Vector& pos, Direction dir) { m_owner = &other; }
+  virtual void ungrab(MovingObject& other, Direction ) { m_owner = nullptr; }
 
+  virtual MovingObject* get_owner() { return m_owner; }
   virtual bool is_portable() const { return true; }
 
   /** Is the object so heavy/bulky/fragile that Tux can't run while
       carrying it? */
   virtual bool is_hampering() const { return false; }
+protected:
+  MovingObject* m_owner = nullptr;
```

Comment:
Is there a reason not to use `MovingObject& m_owner`? If not, inorder for the build not to fail you'd have to add copy constructor +operator= to the `Portable` class (that's the reason the [CI build fails](https://travis-ci.org/SuperTux/supertux/jobs/582162067#L3897)).